README.md

```diff
@@ -3,6 +3,15 @@
 ![logo-solid-bright](https://raw.githubusercontent.com/codingteam/logo/master/logo-solid-bright.png)
 ![logo-solid-dark](https://raw.githubusercontent.com/codingteam/logo/master/logo-solid-dark.png)
 
+## Installation
+
+If you want to use the logo in your project, one way of doing that is to install
+the logo through npm:
+
+```console
+$ npm install codingteam/logo#1.0.0 
```

Comment:
Does this command install the logo locally? Does it require anything else? I tried the command on my machine and it spitted this:

```
npm ERR! git rev-list -n1 1.0.0: fatal: ambiguous argument '1.0.0': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 1.0.0: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 1.0.0: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 1.0.0: 
npm ERR! git rev-list -n1 1.0.0: fatal: ambiguous argument '1.0.0': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 1.0.0: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 1.0.0: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 1.0.0: 
npm ERR! git rev-list -n1 1.0.0: fatal: ambiguous argument '1.0.0': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 1.0.0: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 1.0.0: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 1.0.0: 
npm ERR! Linux 4.4.19
npm ERR! argv "/nix/store/mar2mcqs38jjmazm95gi6v786qaqyldc-nodejs-5.9.0/bin/node" "/home/rexim/.nix-profile/bin/npm" "install" "codingteam/logo#1.0.0"
npm ERR! node v5.9.0
npm ERR! npm  v3.7.3
npm ERR! code 128

npm ERR! Command failed: git rev-list -n1 1.0.0
npm ERR! fatal: ambiguous argument '1.0.0': unknown revision or path not in the working tree.
npm ERR! Use '--' to separate paths from revisions, like this:
npm ERR! 'git <command> [<revision>...] -- [<file>...]'
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/rexim/Programming/codingteam/logo/npm-debug.log
```